_dev/apps/ui/stories/banners-ads-overviews.stories.ts

```diff
@@ -113,7 +113,7 @@ const TemplateCampaignsList = (args, { argTypes }) => ({
 export const BannerOnCampaignsListPage:any = TemplateCampaignsList.bind({});
 BannerOnCampaignsListPage.args = {
   beforeMount() {
-    this.$store.state.campaigns.campaigns.pMaxList = [];
+    this.$store.state.campaigns.campaigns.campaignsList = [];
```

Comment:
By seeing this line, I'm wondering if removing a layer of `campaign` would be interesting